### ISSUE TITLE
Fix `declaration-block-no-redundant-longhand-properties` false negatives for `font-variant`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7620]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
  font-variant-ligatures: foo;
  font-variant-position: foo;
  font-variant-caps: foo;
  font-variant-numeric: foo;
  font-variant-alternates: foo;
  font-variant-east-asian: foo;
  font-variant-emoji: foo;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": {
    "declaration-block-no-redundant-longhand-properties": true
  },
  "fix": true
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdugLgMwkxwFoA3JAJwEskjiAbKgcyRwFcKYBneFAiANxZcBOuWq0SABwhcqOKoV78h2fIRLiadMEik8+EQcPVjK2kpjYBbGNTDKjqkRrLnJxJPRx3Mrbo7GaqKa7nQwSFwkkdqBzqahEuHWEABWVHFYAL5YIAA0IHhU9DAAcki2CCAwAB4VUiUAdGBcXPnghEVMVRiYKCjoIBRsJVyDvL39-YOwYPSUrIqYxABG9BBgANbEmBDEnFBsmFAe65hMABa0UMRSFBBSdgrc4yg4wzBY-TmYWe2QmC6ADEIBRrKwqqkuIR2rB9D0vgMQFEAJ4lRhEV6DeY+KKDbIgLJAA)

### What did you expect to happen?

```css
a {
  font-variant: foo foo foo foo foo foo foo;
}
```

ref: https://www.w3.org/TR/css-fonts-4/#font-variant-prop

### Do you have a proposal to fix the bug?

update `properties.mjs`
